src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --focus_color: #00acdc;
+}
+
+html {
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.main_header {
+  display: flex;
+  width: 100%;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  margin: 0;
+  padding-inline-start: 0;
+  display: flex;
+  list-style: none;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.list__item:nth-child(n + 2) {
+  margin-left: 20px;
+}
+
+.link_list_item {
+  white-space: nowrap;
+  padding: 22.5px 0;
+  display: flex;
+  background-color: #fff;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  line-height: 14px;
+}
+
+.link_list_item:hover {
+  color: var(--focus_color);
+}
+
+.logo_link {
+  height: 40px;
+  width: 40px;
+  padding-top: 10px;
+}
+
+.is-active {
+  position: relative;
+  color: var(--focus_color);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  top: 56px;
+  height: 4px;
+  width: 100%;
+  background-color: var(--focus_color);
+  border-radius: 8px;
+}
```

Comment:
It's not good to specify constant values for such things. If you change the height of the navbar it's going to brake.

What you could do instead is:
1. Move the height of this element into the variable.
2. Calculate the top in this way: `calc(100% - var(--is-active_underline_height))`.